reconcile/utils/terrascript_aws_client.py

```diff
@@ -4900,6 +4901,81 @@ def populate_tf_resource_elasticsearch(self, spec):
                     )
                 )
         # ++++++++ END: REMOVE ++++++++++
+        # add master user creds to output and secretsmanager if internal_user_database_enabled
+        if advanced_security_options:
+            if advanced_security_options.get("internal_user_database_enabled", False):
```

Comment:
Nested ifs can be an and. That saves us one indent level.

Reply:
I actually can't use a nested if here, pushed a commit to move some of the code around and make it clearer